src/storage.py

```diff
@@ -106,6 +120,237 @@ def delete_data(self):
         self._labels_valence = None
         self._found_inds = []
 
+    def sliding_window_train(self, sliding_window, prior=np.array([])):
```

Comment:
Warning message: 
FutureWarning: Feature names only support names that are all strings. Got feature names with dtypes: ['int', 'str']. An error will be raised in 1.2.

Reply:
Problem wasn't the feature names, even when I changed to strings another warning came from sklearn. Fixed by passing in X.values instead of X when training a regressor.